### DISCUSSION TITLE
Poche pretese...

### DISCUSSION THREAD

**Discussion by Miksca** [indomus/forum#3265] [HUB: Homebridge & Apple HomeKit]
Ciao ho letto le vostre guide e vi faccio i complimenti per la passione e la professionalità di questo sito.
Io ho un obiettivo abbastanza semplice:
vorrei, con la sola app di apple home kit senza altri programmi tipo homebrige ec.. gestire:
Telecamere a batteria per interni
telecamere a batteria per esterni
sensori di perdita acqua
sensori per fumo

Null'altro! ho a disposizione un apple tv  che farebbe da hub e vorrei una soluzione semplice ma affidaile.
Qualche consiglio?

**Comment by Scialla** [comment]
Se la AppleTV è almeno una 4K seconda gen. ha Tread. Puoi comprarti dispositivi compatibili Matter, anche Tread.
Oppure prendili direttamente compatibili HomeKit (se li trovi)
Altre soluzioni sono quelle che NON vuoi adottare

**Comment by Miksca** [comment]
Ok grazie

**Comment by KC-inDomus** [comment]
> vorrei, con la sola app di apple home kit senza altri programmi tipo homebrige ec.. gestire: 

te ne pentirai da 3, 2, 1....

**Reply by Miksca** [reply]
capito.... ! qualcosa di facile? Homey?

**Comment by Miksca** [comment]
Ho riletto un po le vostre guide e mi sono deciso per Home assistant
ho ordinato questo [mini pc in base al bechmark 5645](https://www.amazon.it/gp/product/B0CLCZBVTQ?th=1&linkCode=ll1&tag=indomus-21&linkId=30a3a2bc990365b7533137443dfb0b4d&language=it_IT&ref_=as_li_ss_tl)? va bene?
o meglio [4022](https://www.amazon.it/Beelink-Computer-i3-5005U-Graphics-Supporto/dp/B07RNKFJHT?__mk_it_IT=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=5TNR8QV7A0ZT&keywords=mini+pc&qid=1661840705&refinements=p_n_deal_type%3A26901107031&rnid=26901106031&sprefix=mini+p%2Caps%2C94&sr=8-1&linkCode=ll1&tag=indomus-21&linkId=b3d930d35788e9b296c5b426665f4797&language=it_IT&ref_=as_li_ss_tl)

**Reply by KC-inDomus** [reply]
si è ok

**Comment by Miksca** [comment]
Avete qualche consiglio per UPS per mini PC?
o avete qualche consiglio in caso di interruzione di corrente?
Grazie

**Reply by KC-inDomus** [reply]
https://indomus.it/guide/come-scegliere-un-ups-per-la-propria-domotica-e-altro/
